pass/pass.go

```diff
@@ -87,7 +87,12 @@ func (p Pass) Add(creds *credentials.Credentials) error {
 
 	encoded := base64.URLEncoding.EncodeToString([]byte(creds.ServerURL))
 
-	_, err := p.runPass(creds.Secret, "insert", "-f", "-m", path.Join(PASS_FOLDER, encoded, creds.Username))
+	username := creds.Username
+	if strings.Contains(username, "/") {
+		username = base64.URLEncoding.EncodeToString([]byte(creds.Username))
```

Comment:
Do we know if pass supports spaces in usernames? If it does, perhaps we could swap `"/"` for `" "` (as I don't think we support spaces in usernames, so that would be non-ambiguous).

Advantage would be that the username would still be readable (I don't use `pass` myself, but ISTR there's tools to manage credentials in `pass`, which would otherwise see a base64-encoded username (which is not very user-friendly 😅)